Terminal.Gui/ConsoleDrivers/FakeDriver/FakeDriver.cs

```diff
@@ -270,111 +256,39 @@ Key MapKey (ConsoleKeyInfo keyInfo)
 				return Key.Unknown;
 			}
 
-			return (Key)((uint)keyInfo.KeyChar);
+			return ConsoleKeyMapping.MapKeyModifiers (keyInfo, (Key)((uint)keyInfo.KeyChar));
 		}
 
 		var key = keyInfo.Key;
 		if (key >= ConsoleKey.A && key <= ConsoleKey.Z) {
 			var delta = key - ConsoleKey.A;
-			if (keyInfo.Modifiers == ConsoleModifiers.Control) {
-				return (Key)(((uint)Key.CtrlMask) | ((uint)Key.A + delta));
-			}
-			if (keyInfo.Modifiers == ConsoleModifiers.Alt) {
-				return (Key)(((uint)Key.AltMask) | ((uint)Key.A + delta));
```

Comment:
You must add this code again for the `QuitKey_Getter_Setter` unit test pass. The reason is Ctrl-Key must use the chars from '\u0001' through '\u001a' to get the right upper case letter. It's how `ConsoleKeyInfo` works. e.g. To press Ctrl-Q you must send the follow:
`Application.Driver.SendKeys ('\u0011', ConsoleKey.Q, false, false, true);`
So, `A=65` and `Q=81`, `delta=81-65=16` and `65+16=81`.